pallets/did/src/lib.rs

```diff
@@ -292,6 +343,74 @@ impl DidDetails {
 	}
 }
 
+/// Generates a new DID entry starting from the current one stored in the
+/// storage and by applying the changes in the DIDUpdateOperation. The operation
+/// fails with a DidError if the update operation instructs to delete a
+/// verification key that is not associated with the DID or if the operation
+/// counter is not larger than the one stored on chain. !!! Please note that
```

Comment:
Generally I would like to move notes like this to a new line and not use exclamation marks as suggested previously. This is handled similarly in Substrate.

```suggestion
/// counter is not larger than the one stored on chain. 
/// 
/// Please note that
```